### DISCUSSION TITLE
PA-RISC Support?

### DISCUSSION THREAD

**Discussion by Oichkatzelesfrettschen** [kth5/archpower#73] [Ideas]
I have 2x HP c8000 workstations with 32GB RAM. I can donate one if you need. Hoping for PA-RISC support for these to run modern software on older hardware.

**Comment by kth5** [comment]
As much as I would love to say yes, my wife would probably kill me when she sees one more machine under my desk. Really ought to tidy it up and take a picture one of these days...

All joking aside, do these have a BMC? Might have a go but to be honest, I wouldn't port much more than the `base` and `base-devel`  package groups with the correct boot glue since there isn't all that much demand for running desktop environments on PA-RISC these days, is there?
